src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py

```diff
@@ -349,6 +349,7 @@
 helps['vmss stop'] = """
     type: command
     short-summary: Power off (stop) VMs within a VMSS.
+    long-summary: The VMs will continue to be billed. To deallocate VMs within a VMSS, see "az vmss deallocate"
```

Comment:
Maybe more explicit, like `The VMSS will continue to be billed. To avoid that, you can deallocate it through "az vmss deallocate"`?

Reply:
Sounds good to me, will update.